.gitignore

```diff
@@ -1,3 +1,7 @@
 report.sh
 /sippy
 /data
+!sippy-ng/build/.gitignore
+sippy-ng/build/
```

Comment:
Nit, I would put this in the gitignore in sippy-ng

Reply:
Yeah, I think what I really needed was to run make clean to remove the extraneous sippy-ng/build files and not add to .gitignore